searchlight_ui/dashboards/search/search/urls.py

```diff
@@ -15,10 +15,10 @@
 from django.conf.urls import patterns
 from django.conf.urls import url
 
-from searchlight_ui.dashboards.project.search import views
+from searchlight_ui.dashboards.search.search import views
```

Comment:
follows dashboard.panel pattern for now until we find a way to make panel display as dashboard.